10_code/70_FL_and_controls_pop_merge.py

```diff
@@ -0,0 +1,87 @@
+import pandas as pd
```

Comment:
The same comments from 10_code/60_WA_and_controls_pop_merge.py can apply to this one